pyspark_ai/spark_sql_chain.py

```diff
@@ -29,24 +30,28 @@ def run(
     ) -> str:
         assert not args, "The chain expected no arguments"
         # assert llm is an instance of BaseChatModel
-        assert isinstance(
-            self.llm, BaseChatModel
-        ), "The llm is not an instance of BaseChatModel"
+        #assert isinstance(
+        #    self.llm, BaseChatModel
```

Comment:
I note both `BaseChatModel` & `BaseLLM` extend `BaseLanguageModel`. Maybe, here we can keep the check, but just use `BaseLanguageModel` to check.